hack/test-cmd.sh

```diff
@@ -147,6 +147,18 @@ openshift start \
   --etcd-dir="${ETCD_DATA_DIR}" \
   --images="${USE_IMAGES}"
 
+# validate config that was generated
+[ "$(oadm validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml 2>&1 | grep SUCCESS)" ]
```

Comment:
Like diagnostics when it graduates from ex is probably more "validate cluster" and other pieces